common.go

```diff
@@ -381,3 +383,29 @@ func IsInf(f float32, sign int) bool {
 func MissingArgument(field string) error {
 	return fmt.Errorf("missing required configuration option `%s'", field)
 }
+
+// Make GetCertPool a common function.
+func GetCertPool(path string) (*x509.CertPool, error) {
```

Comment:
Go automatically documents public functions (Capitalized function name) when the comment above it starts with the same name. Please change the comment to start with "// GetCertPool [...]", e.g. `// GetCertPool accepts a path to a directory of certificate authorities to trust. Pass in the empty string to use system defaults`